src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -59,10 +81,21 @@ export class SignifyApi {
     this.opRetryInterval = opRetryInterval;
   }
 
+  async bootAndConnect(retryInterval = 1000) {
+    try {
+      await this.signifyClient.boot();
+      await this.signifyClient.connect();
```

Comment:
Hmm only noticed this now but we should only call boot once. This is why we always call connect and if this fails we call boot and connect.

What happens if we call boot with a Signify bran that has already been booted before?

Reply:
It just responds with the "bad request" status. The execution code flow is not affected because it does not throw any error.

Reply:
I think we can keep it incase the `boot` call got failed.

Reply:
ok lets leave it in this case. The main reason I only call connect first in the original flow is to avoid calling boot every time. resolving